### DISCUSSION TITLE
Unit 1.4 quizzes - Latex not rendering on Chrome

### DISCUSSION THREAD

**Discussion by angellmethod** [Lightning-AI/dl-fundamentals#2] [Lecture videos and quizzes]
For example in Quiz 1.4 part 3, it shows as:

$ x_{3}^{[5]} $ 

(I added spaces since github rendered this correctly). This is true for part 4, 5 and 6 as well.

**Comment by rasbt** [comment]
Can conform, and thanks for the note, looks like the mathjax integration is not working like you said. Will ping our web dev!

**Reply by rasbt** [reply]
A little update, our webdev is working on it!

**Comment by rasbt** [comment]
I think this has been fixed a while back. Closing to do some spring cleaning here :)
